### DISCUSSION TITLE
Seeking Guidance on Using AeroSandbox as a Newcomer

### DISCUSSION THREAD

**Discussion by AngLach** [peterdsharpe/AeroSandbox#108] [Q&A] [answered]
I'm currently an undergraduate student majoring in Mechanical and Aerospace Engineering. I recently stumbled upon your impressive aerodynamics project while searching for a VLM code to study multiple wings within a custom script that I've been working on.

I am relatively new to programming, and this is my first attempt at working with such a large project. The reason I reached out to your project is that I've been looking for a solution to efficiently analyze various wing geometries. While I have some experience with XFLR5, I've found that it can be time-consuming to run simulations for different wing configurations. Also my experience with computational methods for solving aerodynamic problems is limited to 2d airfoils ( for example discrete vortex method from Katz/Plotkin). 

Your project appears to be a valuable resource for my aerodynamics studies, and I'm eager to learn how to use it effectively. However, I'm seeking some guidance on getting started.
The first step that i would like to achieve is to use from the aerodynamics tutorial the vlm script in an iterative process for different wing geometries.
 
I'm enthusiastic about diving into the world of aerodynamics and programming, and I genuinely appreciate any guidance or pointers you can provide to help me make the most of your project. I'm eager to contribute to this community and learn from experienced users like yourselves.

Thank you for your time and assistance.

Best regards,
DecksDark01

**Comment by kjayawar** [comment, marked as answer]
HI Decks,

You can check the tutorial section on the repo to begin with I'd suggest.
